src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -3171,6 +3178,320 @@ <h2>{{getValue "notes-page-count"}}. Debtors</h2>
 
           {{end}}
           <!-- DEBTORS NOTE END -->
+          
+          <!-- CREDITORS WITHIN ONE YEAR NOTE START -->
+          {{if $creditorsWithinOneYearNote := $balanceSheetNotes.creditors_within_one_year}}
+            {{incValue "notes-page-count" }}
+            {{setValue "creditors-within-one-year-notes-id" (getValue "notes-page-count") }}
+
+            <div class="page">
+              <div class="header">
+                <h1 class="text--center">
+                  <ix:nonNumeric contextRef="CY" name="uk-bus:EntityCurrentLegalOrRegisteredName">{{$companyName}}</ix:nonNumeric>
+                </h1>
+              </div>
+
+              <h2 class="text--center">Notes to the Financial Statements</h2>
+              <p class="text--center">
+                <strong>for the Period Ended {{$period.current_period_end_on_formatted}}</strong>
+              </p>
+
+              <h2>{{getValue "notes-page-count"}}. Creditors: amounts falling due within one year note</h2>
+
+              <table cellspacing="10" id="creditors-within-one-year-note-table">
+                <tbody>
+
+                <tr>
+                  <th class="table-heading"></th>
+                  {{if $creditorsWithinOneYearNote.total.current_amount}}
+                    <th class="table-heading" id="creditors-within-one-year-note-current-heading">{{$period.current_period_bs_date}}</th>
+                  {{else}}
+                    <th  class="table-heading">&#160;</th>
+                  {{end}}
+
+                  {{if $creditorsWithinOneYearNote.total.previous_amount}}
+                    <th class="table-heading" id="creditors-within-one-year-note-previous-heading">{{$period.previous_period_bs_date}}</th>
+                  {{end}}
+                </tr>
+
+                <tr>
+                  <th class="table-heading"></th>
+                  {{if $creditorsWithinOneYearNote.total.current_amount}}
+                    <th class="table-heading" id="creditors-within-one-year-note-current-currency">£</th>
+                  {{else}}
+                    <th  class="table-heading">&#160;</th>
+                  {{end}}
+
+                  {{if $creditorsWithinOneYearNote.total.previous_amount}}
+                  <th class="table-heading" id="creditors-within-one-year-note-previous-currency">£</th>
+                  {{end}}
+                </tr>
+
+                {{if $bankLoansAndOverdrafts := $creditorsWithinOneYearNote.bank_loans_and_overdrafts}}
+                  <tr class="figures">
+                    <td id="bank-loans-and-overdrafts">Bank loans and overdrafts</td>
```

Comment:
Can we please rename one of the ids for `id="bank-loans-and-overdrafts"`? as it is duplicated adn id's should be unique